apps/src/lib/levelbuilder/lesson-editor/StandardsEditor.jsx

```diff
@@ -156,7 +156,11 @@ function StandardsEditor(props) {
       <label>
         <strong>Filter by framework</strong>
       </label>
-      <select onChange={handleSelectFramework} style={styles.select}>
+      <select
+        onChange={handleSelectFramework}
+        style={styles.select}
+        aria-label="Filter by framework"
```

Comment:
Not translating since this is in an internal tool where none of the other strings are translated. Maybe someday we'll support level building in languages other than English, but not at the moment!